### ISSUE TITLE
Public: Change Blacklight favicon to UCLA icon

### ISSUE THREAD

**Issue by little9** [UCLALibrary/ursus#26]
When loading the home page, the Blacklight favicon should be the UCLA one and not the Blacklight default. 

Child of UCLALibrary/amalgamated-samvera#129

**Comment by andrewbenedictwallace** [comment]
![Screen Shot 2018-10-17 at 3.43.01 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5aecc25f15dd020013d072cb/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b76236455f46b7191a37bf116284f5f5e5ecc3c4a4356bcbbd671f3e57862804e0ebfd2380b75c23518fdb88b4eea01ba21d9498cb1ba8b49d2f42f67b6d5db1063474f4dcbed5727a19e41c95df542e94c53a6133763ee7e2431.png)
